### PR TITLE
Figures for kCSD paper

### PR DESCRIPTION
**Pull request by m-kowalska**
Modified style of figures, added README, changed names of scripts and folder with figures.

### CODE REVIEW

**Review by wsredniawa — APPROVED** [review]

**Review by wsredniawa — APPROVED** [review]

**Review by wsredniawa — APPROVED** [review]

**Review by wsredniawa — APPROVED** [review]

File: kcsd/validation/VisibilityMap.py [hunk 1]
```diff
@@ -553,7 +726,7 @@ def plot_error_map(self, point_error, ele_pos):
                         true_csd_xlims=TRUE_CSD_XLIMS, sigma=0.3,
                         src_type='gauss', n_src_init=100, ext_x=0.1)
     rms, point_error = k.calculate_error_map(CSD_PROFILE,
-                                             Rs=np.arange(0.2, 0.5, 0.1))
+                                              Rs=np.arange(0.2, 0.5, 0.1))
```
Comment (abukaj) [inline kcsd/validation/VisibilityMap.py:729]: Is that change necessary? (IMO it introduces horizontal misalignment).

File: kcsd/validation/VisibilityMap.py [hunk 2]
```diff
@@ -562,8 +735,18 @@ def plot_error_map(self, point_error, ele_pos):
                         ele_lims=ELE_LIMS, true_csd_xlims=TRUE_CSD_XLIMS,
                         est_xres=0.01, est_yres=0.01)
     rms, point_error = a.calculate_error_map(CSD_PROFILE,
-                                             Rs=np.arange(0.05, 0.5, 0.05),
-                                             lambdas=np.array(0), n=2)
+                                              Rs=np.arange(0.05, 0.5, 0.05),
+                                              lambdas=np.array(0), n=2)
```
Comment (abukaj) [inline kcsd/validation/VisibilityMap.py:739]: Is that change necessary? (IMO it introduces horizontal misalignment).

File: kcsd/validation/VisibilityMap.py [hunk 3]
```diff
@@ -562,8 +735,18 @@ def plot_error_map(self, point_error, ele_pos):
...
+
+    print('Checking 2D MoI')
+    ELE_LIMS = [0.118, 0.882]
+    CSD_PROFILE = CSD.gauss_2d_large
+    a = VisibilityMap2DMoI(total_ele=9, h=50., sigma=1., n_src_init=1000,
+                            ele_lims=ELE_LIMS, true_csd_xlims=TRUE_CSD_XLIMS,
+                            est_xres=0.01, est_yres=0.01)
```
Comment (abukaj) [inline kcsd/validation/VisibilityMap.py:746]: I see a pattern here.  But it is inconsistent with horizontal alignment of call arguments used in the project.

File: kcsd/validation/VisibilityMap.py [hunk 4]
```diff
@@ -562,8 +735,18 @@ def plot_error_map(self, point_error, ele_pos):
...
+    rms, point_error = a.calculate_error_map(CSD_PROFILE,
+                                              Rs=np.arange(0.05, 0.5, 0.05),
+                                              lambdas=np.array(0), n=2)
```
Comment (abukaj) [inline kcsd/validation/VisibilityMap.py:749]: Horizontal alignment of call arguments inconsistent with used in the project.

File: figures/what_you_can_see_with_lfp/npx/Fig3-figure_traub_timespace.py [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+
+def set_axis(ax, letter=None):
+    ax.text(
+        -0.05,
+        1.05,
+        letter,
+        fontsize=20,
+        weight='bold',
+        transform=ax.transAxes)
```
Comment (abukaj) [inline figures/what_you_can_see_with_lfp/npx/Fig3-figure_traub_timespace.py:21]: Call formatting inconsistent with used in the project. Is there a reason for that?

File: figures/npx/traub_data_kcsd_column_figure.py [hunk 6]
```diff
@@ -187,7 +188,7 @@ def extract_csd_timepoint(h, pop_names, time_pts, field_name):
         src = fetch_mid_pts(h, pop_name)
         print()
         idx = find_map_idx(h, pop_name, field_name)
-        src_time = h['/data/uniform/' + pop_name + '/' + field_name].value
+        src_time = h['/data/uniform/' + pop_name + '/' + field_name][()]
```
Comment (abukaj) [inline figures/npx/traub_data_kcsd_column_figure.py:191]: I do not understand what happens here.

File: figures/npx/traub_data_kcsd_column_figure.py [hunk 7]
```diff
@@ -208,6 +209,7 @@ def extract_csd_timepoint(h, pop_names, time_pts, field_name):
     return np.array(all_x), np.array(all_y), np.array(all_z), np.array(all_val)
 
 
+
```
Comment (abukaj) [inline figures/npx/traub_data_kcsd_column_figure.py:212]: Is that newline necessary? See: https://pep8.org/#blank-lines

If it separates groups of related functions, a comment would be appreciated.

**Review by abukaj — APPROVED** [review]
Indents look fine now. ;)
